cpp/ql/lib/DefaultOptions.qll

```diff
@@ -62,7 +62,7 @@ class Options extends string {
     f.getASpecifier().hasName("noreturn")
     or
     f.hasGlobalOrStdName([
-        "exit", "_exit", "abort", "__assert_fail", "longjmp", "__builtin_unreachable"
+        "exit", "_exit", "_Exit", "abort", "__assert_fail", "longjmp", "__builtin_unreachable"
```

Comment:
I'm curious why "longjmp" is on this list.  It's not the same as the others.  I suppose it exits the current position in code, never to return to where it left off.

(I don't expect you to have an answer)

Reply:
Indeed, I don't have an answer.

Reply:
I agree that we should probably rename the predicate. I'm also a bit hesitant to do so given that it lives in  `DefaultOptions.qll`, so it's a bit difficult to gauge the impact of this outside of the CodeQL repo.